### PR TITLE
GIX-1111: List SNS Proposal Topics

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Add method to retrieve the Nervous System Functions from an SNS project.

Those are the "topics" that a neuron can follow.

# Changes

* Add listNervousSystemFunctions method to sns governance.
* Add listNervousSystemFunctions method to sns wrapper.
* Export ListNervousSystemFunctionsResponse and NervousSystemFunction types.

# Tests

* Test new sns wrapper method.
* Test new sns governance method.

### CODE REVIEW

**Comment by lmuntaner** [comment]
@mstrasinskis @peterpeterparker please review

I tested it and I got the list of functions.

File: packages/sns/src/sns.wrapper.ts [hunk 1]
```diff
@@ -108,6 +108,9 @@ export class SnsWrapper {
     params: Omit<SnsListNeuronsParams, "certified">
   ): Promise<Neuron[]> => this.governance.listNeurons(this.mergeParams(params));
 
+  listNervousSystemFunctions = () =>
```
Comment (peterpeterparker) [inline packages/sns/src/sns.wrapper.ts:111]: - could you add `... = (params: Omit<QueryParams, "certified">) => ...` to follow the other function pattern?

- could you type the return type?

Comment (peterpeterparker) [reply packages/sns/src/sns.wrapper.ts:111]: or it's `Promise<void>`?

Comment (lmuntaner) [reply packages/sns/src/sns.wrapper.ts:111]: Sure

File: packages/sns/README.md [hunk 2]
```diff
@@ -162,6 +163,15 @@ List the neurons of the Sns
 | ------------- | ----------------------------------------------------- |
 | `listNeurons` | `(params: SnsListNeuronsParams) => Promise<Neuron[]>` |
 
+##### :gear: listNervousSystemFunctions
+
+List Nervous System Functions
+Neurons can follow other neurons in specific Nervous System Functions.
```
[the remark this answers is not in the document]
Comment (lmuntaner) [reply packages/sns/README.md:169]: https://github.com/dfinity/ic/blob/master/rs/sns/governance/proto/ic_sns_governance/pb/v1/governance.proto#L197

> A NervousSystem function that can be executed by governance as a result of an adopted proposal.

Which are then used in the followees:

https://github.com/dfinity/ic/blob/master/rs/sns/governance/proto/ic_sns_governance/pb/v1/governance.proto#L142

File: packages/sns/src/sns.wrapper.ts (hunk 1 above)
Comment (lmuntaner) [reply packages/sns/src/sns.wrapper.ts]: done

**Comment by lmuntaner** [comment]
@peterpeterparker ready for another review

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

**Pull request merged**